project/Dependencies.scala

```diff
@@ -4,18 +4,19 @@ import Keys._
 object Dependencies {
 
   object Versions {
-    val circeV        = "0.9.0-M3"
-    val catsV         = "1.0.0-RC2"
-    val catsEffV      = "0.6"
+    val circeV        = "0.9.0"
+    val catsV         = "1.0.1"
+    val catsEffV      = "0.7"
```

Comment:
cats-effect 0.8 targets 1.0.1

Reply:
Okay, didn't want to update it since both Http4s and Fs2 versions are based on Cats Effect 0.7, but I guess they are compatible so no harm.

Reply:
If all tests pass it should be ok. I'm mostly trying to avoid the whole versioning issue